tests/ui/test_ui.py

```diff
@@ -81,6 +81,35 @@ def test_set_footer_text_with_duration(self, view, mocker,
         mock_sleep.assert_called_once_with(duration)
         assert view.controller.update_screen.call_count == 2
 
+    @pytest.mark.parametrize('state, footer_text, num_suggestions', [
+        (0, [('code', ' help '), ' community ', ' issues '], 3),
```

Comment:
If you're going to test the actual text, can we extract the style as a parameter at least?

Reply:
Good idea; I can use the fixture from `conftest.py` too.